js/src/views/Accounts/List/list.css

```diff
@@ -14,26 +14,8 @@
 /* You should have received a copy of the GNU General Public License
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
-.list {
-  display: flex;
-  flex-wrap: wrap;
-}
-
-.item {
-  flex: 0 1 50%;
-  width: 50%;
-  position: relative;
-  padding-bottom: 0.25em;
-  box-sizing: border-box;
-}
 
-.item:nth-child(odd) {
-  padding-right: 0.125em;
-}
-
-.item:nth-child(even) {
-  padding-left: 0.125em;
-}
+@import '../../_list.css';
```

Comment:
Good point, will try. (Should work, honestly was a bit unsure about it since it is CSS)